aws-iam-role-poweruser/variables.tf

```diff
@@ -9,6 +9,12 @@ variable "source_account_id" {
   description = "The source AWS account to establish a trust relationship. Ignored if empty or not provided."
 }
 
+variable "source_account_ids" {
```

Comment:
worth adding a deprecation notice on the source_account_id ?

Reply:
oh also, `make docs`

Reply:
Done on both.